e2e/wdio/core/tests/select.e2e-spec.ts

```diff
@@ -0,0 +1,398 @@
+        });
+
+        it('should be able to add remove option', () => {
+            click(addRemoveOptionExample + buttons, 2);
+            waitForElDisplayed(option, 4);
+            const optionsCountBefore = getElementArrayLength(option);
+            click(addRemoveOptionExample + buttons, 2);
+            click(addRemoveOptionExample + buttons, 1);
+            click(addRemoveOptionExample + buttons, 2);
+            waitForElDisplayed(option, 4);
+            const optionsCountAfterAdding = getElementArrayLength(option);
+
+            expect(optionsCountBefore).toEqual(optionsCountAfterAdding + 1);
+        });
+
+        it('should have visual check', () => {
+            click(addRemoveOptionExample + buttons, 2);
+            waitForElDisplayed(option, 4);
+            const arrLength = getElementArrayLength(option);
+
+            for (let i = 4; arrLength > i; i++) {
+                checkElementStates(option, addRemoveSelectOptions + i, optionTag, i);
+            }
+        });
+    });
+
+    describe('Programmatic Control', function() {
+        it('should be able to select the option', () => {
+            const textBefore = getText(programmaticControlExample + displayedText);
+            click(programmaticControlExample + buttons, 3);
+            waitForElDisplayed(option, 4);
+            click(options(60));
+            const textAfter = getText(programmaticControlExample + displayedText);
+            expect(textBefore).not.toEqual(textAfter);
+        });
+
+        it('should be able to control select by buttons', () => {
+            const textBefore = getText(programmaticControlExample + displayedText);
+            click(programmaticControlExample + buttons, 0);
+            const textAfter = getText(programmaticControlExample + displayedText);
+
+            click(programmaticControlExample + buttons, 1);
+            waitForElDisplayed(option, 4);
+            click(programmaticControlExample + buttons, 2);
+            waitForInvisibilityOf(overlayContainer);
+
+            expect(textBefore).not.toEqual(textAfter);
+        });
+
+        it('should have visual check', () => {
+            click(programmaticControlExample + buttons, 3);
+            waitForElDisplayed(option, 4);
+            const arrLength = getElementArrayLength(option);
+
+            for (let i = 4; arrLength > i; i++) {
+                checkElementStates(option, programmaticSelectOptions + i, optionTag, i);
+            }
+        });
+    });
+
+    describe('Check orientation', function() {
+        it('should check RTL and LTR', () => {
+            selectPage.checkRtlSwitch();
+        });
+    });
+
+    describe('Check visual regression', function() {
+        beforeEach(() => {
+            refreshPage();
+        });
+
+        it('should check examples visual regression', () => {
+            selectPage.saveExampleBaselineScreenshot('select-core');
+            expect(selectPage.compareWithBaseline('select-core')).toEqual(0);
```

Comment:
please change '.toEqual' to 'toBeLessThan' 1 or 2,  for stability

Reply:
Done